build/proj/CoreBuild.proj

```diff
@@ -38,6 +39,14 @@
     <SolutionRestoreAndBuildProperty Include="__DeployProjectOutput=$(Deploy)" Condition="'$(Deploy)' != ''"/>
   </ItemGroup>
 
+  <Target Name="LogInitialMessage">
+
+    <Message
+      Text ="Building $(SolutionFileName) [$(Configuration)]...%0a"
```

Comment:
What's `%0a`? Also there's a space before the `=`.

Reply:
Remove space, it's a newline.